Gemfile

```diff
@@ -30,14 +59,27 @@ group :test do
   gem 'database_cleaner', '~> 1.3.0'
   gem 'simplecov', '~> 0.9'
 end
+>>>>>>> develop
 
 group :development, :test do
   gem 'awesome_print'
   gem 'pry-rails'
 end
 
+
+
 group :development do
   gem 'web-console', '~> 2.0'
+<<<<<<< HEAD
+
+
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+=======
+>>>>>>> develop
   gem 'spring'
+
+
+
+
```

Comment:
This file needs to be revised and fixed. In this state, it'd be impossible to execute any `bundle` command.